### PR TITLE
Noah Amar [Sprint-Challenge-Redux-Smurfs]

### PR DESCRIPTION
**Pull request by Noah-Amar**

### CODE REVIEW

**Comment by shawnantonucci** [comment]
Try to commit more and use more specific terms for that commit

File: Answers.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects? Which method do we use to create a new object while extending the properties of another object?
+.map, .reduce., filter . Object.assign()
 1.  Describe `actions`, `reducers` and the `store` and their role in Redux. What does each piece do? Why is the store known as a 'single source of truth' in a redux application?
+Actions are packets of information that contain an action type and some data that we want associated with that action type. Actions are plain javascript object, or functions that return plain javascript objects. Reducers are pure functions i.e. they don't produce any side effects. They take in two arguments, the current/previous state that they represent, and the action object that is sent to them by the action creator functions. They will never update state directly but will always return a new object. The store is a single JavaScript Object that represents everything that changes in the application. The store contains the state of the application. The store is known as the single source of truth because it is the only place that represents the state of the application.
```
Comment (jaspindersingh83) [inline Answers.md]: This is too wordy . Refer to redux documents
we defined the actions that represent the facts about “what happened” and the reducers that update the state according to those actions.

The Store is the object that brings them together.

File: smurfs/src/actions/index.js [hunk 2]
```diff
@@ -13,3 +24,64 @@
    U - updateSmurf
    D - deleteSmurf
 */
+
+
+const URL = 'http://localhost:3333/smurfs';
+
+export const getSmurfs = () => {
+  const friends = axios.get(`${URL}/get`);
```
Comment (jaspindersingh83) [inline smurfs/src/actions/index.js]: Make it a promise or async/await as the HTTP call is asynchronous

File: smurfs/src/components/App.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react';
 import './App.css';
 import { getSmurfs } from '../actions';
+import { createSmurf } from '../actions';
```
Comment (jaspindersingh83) [inline smurfs/src/components/App.js]: Both createSmurfs and getSmurfs can be in same line

File: smurfs/src/components/App.js [hunk 4]
```diff
@@ -13,13 +15,17 @@ class App extends Component {
...
   render() {
 
 
     return (
       <div>
+        <Form />
       { this.props.smurfs.map(smurf=><div>{smurf.name}</div>)}
+      { this.props.smurfs.map(smurf=><div>{smurf.age}</div>)}
+      { this.props.smurfs.map(smurf=><div>{smurf.height}</div>)}
       </div>
```
Comment (jaspindersingh83) [inline smurfs/src/components/App.js]: make a const smurfs = this.props.smurfs after return statement, rather than writing this.props.smurfs again and again
